src/odemis/gui/comp/text.py

```diff
@@ -329,8 +328,8 @@ def _showDropDown(self, show=True):
         """
         if show:
             size = self.dropdown.GetSize()
-            width, height = self . GetSizeTuple()
-            x, y = self.ClientToScreenXY(0, height)
+            width, height = wx.Window.GetSize(self)
```

Comment:
`self.GetSize()`, or even `self.Size` should work.
Also, `self.ClientToScreen(0, height)` probably works too.